style.css

```diff
@@ -0,0 +1,32 @@
+// неправильный комментарий
+
+.hEad {
+
+}
+
+
+
+/*нет пробелов в комментарии, это ошибка*/
+
+.wrap {
+
+  position: ;
+
+  display: block;
+
+}
+
+.main {color: aliceblue}
+
+.link {
+    color: aliceblue;
+
+    background: antiquewhite;
+  color: aqua;
+
+
+
+
+
+      position:relative;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/test_test/pullRequest?prid=5392485)